metastore/scripts/upgrade/hive/hive-schema-3.0.0.hive.sql

```diff
@@ -0,0 +1,1334 @@
+-- HIVE system db
+
+CREATE DATABASE SYS;
```

Comment:
The JDCBStorageHandler supports Derby (tested) and should support postgres and oracle too (i am still testing those). The only thing the storage handler needs to support a database is to know how to do a limit/offset query. I've added that for more dbs but haven't tested it all.

I cannot create pure views in sys. The JDBCStorageHandler is needed to bring the data into Hive.   Once hive can access it (through JDBCStorageHandler in sys) you can create additional views on top of those. These additional views no longer need to reference the storage handler, that's implicit because they reference the tables in sys.

Just to be clear I want to be able to query the data in Hive - that's the point of this Jira. You should be able to write hive queries against hive's metadata. I could create views in the HMS layer - but that would mean that Hive cannot run queries on those - only the HMS RDBMS would. (Not sure that's what you're suggesting).